backend/Controllers/AuthenticationController.cs

```diff
@@ -8,6 +8,7 @@ namespace CbtBackend.Controllers;
 
 [ApiController]
 [Route("/user/login")]
+[Produces("application/json")]
```

Comment:
since the `/user/login` returns just plain string i don't think this is needed for now

Reply:
This is needed because the response has to be a json according to spec, but a json with only a single string, ie `"jwt.token.string"`

Reply:
Without this attribute what was returned is just `jwt.token.string` without quotations